src/CMakeLists.txt

```diff
@@ -877,6 +877,9 @@ add_subdirectory(chart)
 
 # have some test programs for unit testing, ...
 if(BUILD_TESTS)
+  find_package(CMocka CONFIG REQUIRED)
```

Comment:
`find_package(cmocka 1.1.0 CONFIG REQUIRED)`

Please add a version requirement, it should be at least 1.1.0

Reply:
I would suggest to put the checking for cmocka and the addition of the cmake function for cmocka tests in a separate commit.

Reply:
Added version requirement and made separate commit.
I have put the library check into src/tests/unittests/CMakeLists.txt - do you think this is the right place?